File: ComplexProject/src/app/services/responses/ParticipantResponse.ts
export class ParticipantResponse{
  id: number
  type: String
  eventId: number
  lName: String
  fName: String
  patronymic: String
  email: String
  phone: String

  constructor(id: number, type: String, eventId: number, lName: String, fName: String, patronymic: String, email: String, phone: String) {
    this.id = id;
    this.type = type;
    this.eventId = eventId;
    this.lName = lName;
    this.fName = fName;
    this.patronymic = patronymic;
    this.email = email;
    this.phone = phone;
  }
}

File: Server/src/main/java/com/complex/server/responses/ParticipantResponse.java
package com.complex.server.responses;

import jakarta.persistence.criteria.CriteriaBuilder;

public record ParticipantResponse(
        Long id,
        String type,
        Integer eventId,
        String lName,
        String fName,
        String patronymic,
        String email,
        String phone
) {}
